ui/src/notebooks/components/PipeList.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {FC, useContext} from 'react'
+import Pipe from 'src/notebooks/components/Pipe'
+import {NotebookContext} from 'src/notebooks/context/notebook'
+
+const PipeList: FC = () => {
+  const {id, pipes} = useContext(NotebookContext)
+  const _pipes = pipes.map((_, idx) => (
+    <Pipe idx={idx} key={`pipe-${id}-${idx}`} />
+  ))
+
+  return <>{_pipes}</>
```

Comment:
```suggestion
  return _pipes
```
you can just straight up return arrays now 😸

Reply:
ITS THE FUTURE

Reply:
nope.. not the future yet. tsc is giving me this:
```
Type '() => JSX.Element[]' is not assignable to type 'FC<{}>'
```